ui/src/BulkUploads.js

```diff
@@ -0,0 +1,302 @@
+        this.setState({
+          fileProgress: 1.0,
+          fileUploadSuccess: true,
+          uploadInProgress: false,
+        });
+
+        this.getJobs();
+      });
+  };
+
+  handleClose = (event, reason) => {
+    // Ignore clickaways so that the dialog is modal
+    if (reason === "clickaway") {
+      return;
+    }
+
+    this.setState({
+      fileUploadSuccess: false,
+    });
+  };
+
+  getJobs = async () => {
+    // TODO: Job Type Temp hardcoded to BULD_REFUSAL
+    const response = await fetch(
+      `/api/job?collectionExercise=${this.props.collectionExerciseId}&jobType=BULK_REFUSAL`
+    );
+
+    // TODO: We need more elegant error handling throughout the whole application, but this will at least protect temporarily
+    if (!response.ok) {
+      return;
+    }
+
+    const jobsJson = await response.json();
+
+    this.setState({ jobs: jobsJson });
+  };
+
+  handleOpenDetails = (job) => {
+    this.setState({ showDetails: true, selectedJob: job.id });
+  };
+
+  handleClosedDetails = () => {
+    this.setState({ showDetails: false });
+  };
+
+  onProcessJob = () => {
+    fetch(`/api/job/${this.state.selectedJob}/process`, {
+      method: "POST",
+    });
+  };
+
+  onCancelJob = () => {
+    fetch(`/api/job/${this.state.selectedJob}/cancel`, {
+      method: "POST",
+    });
+  };
+
+  render() {
+    const selectedJob = this.state.jobs.find(
+      (job) => job.id === this.state.selectedJob
+    );
+
+    const bulkRefusalTableRows = this.state.jobs.map(
+      (job, index) =>
+        this.state.authorisedActivities.includes(
+          "VIEW_BULK_REFUSAL_PROGRESS"
+        ) && (
+          <TableRow key={job.createdAt}>
+            <TableCell component="th" scope="row">
+              {job.fileName}
+            </TableCell>
+            <TableCell>{job.createdAt}</TableCell>
+            <TableCell align="right">
+              <Button
+                onClick={() => this.handleOpenDetails(job)}
+                variant="contained"
+              >
+                {convertStatusText(job.jobStatus)}{" "}
+                {[
+                  "STAGING_IN_PROGRESS",
+                  "VALIDATION_IN_PROGRESS",
+                  "PROCESSING_IN_PROGRESS",
+                ].includes(job.jobStatus) && (
+                  <CircularProgress size={15} style={{ marginLeft: 10 }} />
+                )}
+              </Button>
+            </TableCell>
+          </TableRow>
+        )
+    );
+
+    return (
+      <div style={{ padding: 20 }}>
+        <Link
+          to={`/collex?surveyId=${this.props.surveyId}&collexId=${this.props.collectionExerciseId}`}
+        >
+          ← Back to collection exercise details
+        </Link>
+        <Typography variant="h4" color="inherit">
+          Uploaded Bulk Process Files
+        </Typography>
+        <Typography variant="h6" color="inherit" style={{ marginTop: 20 }}>
+          Bulk Refusals
+        </Typography>
+        <TableContainer component={Paper}>
```

Comment:
this table should not display if the user doesn't have `VIEW_BULK_REFUSAL_PROGRESS` permission